src/Associations.cc

```diff
@@ -250,7 +250,7 @@ void Associations::collectRefStars(afw::table::SimpleCatalog &refCat, geom::Angl
 
     // TODO: we aren't getting covariances from Gaia yet, so maybe ignore this for now?
     try {
-        pmRaDecCovKey = refCat.getSchema().find<float>("pm_ra_Dec_Cov").key;
+        pmRaDecCovKey = refCat.getSchema().find<float>("pm_dec_pm_ra_Cov").key;
```

Comment:
```suggestion
        pmRaDecCovKey = refCat.getSchema().find<float>("pm_ra_pm_dec_Cov").key;
```

Reply:
Thanks for catching this. There is no test coverage of this, probably because `jointcal` doesn't actually do anything with `raDecCov`.